ontology-engine/graph-engine_2.11/src/main/scala/org/sunbird/graph/external/ExternalPropsManager.scala

```diff
@@ -28,6 +28,14 @@ object ExternalPropsManager {
         store.read(request.get("identifier").asInstanceOf[String], fields, getPropsDataType(schemaName, version))
     }
 
+    def deleteProps(request: Request, fields: List[String])(implicit ec: ExecutionContext): Future[Response] = {
+        val schemaName: String = request.getContext.get("schemaName").asInstanceOf[String]
+        val version: String = request.getContext.get("version").asInstanceOf[String]
+        val primaryKey: util.List[String] = SchemaValidatorFactory.getExternalPrimaryKey(schemaName, version)
+        val store = ExternalStoreFactory.getExternalStore(SchemaValidatorFactory.getExternalStoreName(schemaName, version), primaryKey)
+        store.delete(request.get("identifier").asInstanceOf[String], if (fields.nonEmpty) fields else List())
```

Comment:
No need of passing fields, as row can be deleted using primary key. Please remove this implementation.

Reply:
Done